src/__snapshots__/core.test.ts.snap

```diff
@@ -69,6 +69,20 @@ exports[`Core > disablePerLine 1`] = `
 let a = 1;"
 `;
 
+exports[`Core > fixes problems with legacy config and 3rd-party plugins 1`] = `
+"// eslint-disable-next-line test/ban-nullish-coalescing-operator
+// eslint-disable-next-line test/ban-nullish-coalescing-operator
+// eslint-disable-next-line test/ban-nullish-coalescing-operator
+// eslint-disable-next-line test/ban-nullish-coalescing-operator
+// eslint-disable-next-line test/ban-nullish-coalescing-operator
+// eslint-disable-next-line test/ban-nullish-coalescing-operator
+// eslint-disable-next-line test/ban-nullish-coalescing-operator
+// eslint-disable-next-line test/ban-nullish-coalescing-operator
+// eslint-disable-next-line test/ban-nullish-coalescing-operator
+// eslint-disable-next-line test/ban-nullish-coalescing-operator
```

Comment:
I was expecting only one disable comment to be added, but multiple ones were added.

To investigate this problem, I added the following code for the debug log.

- https://github.com/mizdra/eslint-interactive/pull/366/commits/33de5de58245f03f45ffcc1cb59d5c3c5450be23

```console
$ pnpm run test -t "fixes problems" src/core.test.ts
...
unfilteredMessages [
  {
    ruleId: 'test/ban-nullish-coalescing-operator',
    message: "Definition for rule 'test/ban-nullish-coalescing-operator' was not found.",
    line: 1,
    column: 1,
    endLine: 1,
    endColumn: 2,
    severity: 2,
    nodeType: null
  }
]
```

It seems that the plugin is failing to load.

Reply:
`core.lint()` only uses the `ESLint` class. On the other hand, `core.disablePerLine()` uses both the `ESLint` class and the `Linter` class.

The `ESLint` class seems to be able to load 3rd-party plugins, but the `Linter` class cannot.

As a result, it seems that although the lint results can be output, the problems cannot be fixed.